pkg/sdk/event_tables_impl_gen.go

```diff
@@ -0,0 +1,166 @@
+package sdk
+
+import (
+	"context"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/internal/collections"
+)
+
+var _ EventTables = (*eventTables)(nil)
+
+type eventTables struct {
+	client *Client
+}
+
+func (v *eventTables) Create(ctx context.Context, request *CreateEventTableRequest) error {
+	opts := request.toOpts()
+	return validateAndExec(v.client, ctx, opts)
+}
+
+func (v *eventTables) Show(ctx context.Context, request *ShowEventTableRequest) ([]EventTable, error) {
+	opts := request.toOpts()
+	dbRows, err := validateAndQuery[eventTableRow](v.client, ctx, opts)
+	if err != nil {
+		return nil, err
+	}
+	resultList := convertRows[eventTableRow, EventTable](dbRows)
+	return resultList, nil
+}
+
+func (v *eventTables) ShowByID(ctx context.Context, id SchemaObjectIdentifier) (*EventTable, error) {
```

Comment:
Please use `In` syntax here. The current implementation can lead to problems like #2173

Reply:
done